src/l500/l500-fw-update-device.h

```diff
@@ -10,6 +10,9 @@ namespace librealsense
     class l500_update_device : public update_device
     {
     public:
+        static const uint16_t DFU_VERSION_MASK = 0xFE;
```

Comment:
Why not just define these in the function that uses them?

Reply:
I wanted to attach the specific data as much to the specific device. can be moved.